libs/community/langchain_community/embeddings/baidu_qianfan_endpoint.py

```diff
@@ -91,16 +96,25 @@ def validate_environment(cls, values: Dict) -> Dict:
         try:
             import qianfan
 
+            default_values = {
+                name: field.default
+                for name, field in cls.__fields__.items()
```

Comment:
Let's hold off on doing this in more places since it also introduces incompatibility with pydantic 1 

I'm exploring resolving this using a decorator that automatically populates defaults: https://github.com/langchain-ai/langchain/pull/23841

Reply:
Okay, then I'll close this pr.